api/v1beta1/cinderscheduler_types.go

```diff
@@ -28,6 +28,8 @@ type CinderSchedulerTemplate struct {
 
 	// +kubebuilder:validation:Optional
 	// +kubebuilder:default=1
+	// +kubebuilder:validation:Maximum=32
```

Comment:
We will likely want to limit the deployment to a single scheduler. Tests have shown a single scheduler can handle the load, and having only one scheduler will avoid a class of quota bugs that arise when multiple schedulers are processing volume requests in parallel. I'd prefer to set the maximum to 1, and we can raise it later if a single scheduler can't handle the load.

Reply:
One concern with enforcing maximum 1 is that we have to release a new operator to allow users to do scale out if they need it.

However probably c-vol would become bottleneck even if we scale scheduler pods and we unlikely need to scale only scheduler.

Reply:
Updated the value accordingly.